src/common/imageio_dng.h

```diff
@@ -173,12 +174,12 @@ static inline void dt_imageio_dng_write_tiff_header(
   memcpy(buf+400, xtrans, sizeof(uint8_t)*36);
 
   // ColorMatrix1 try to get camera matrix else m[k] like before
-  XYZ_CAM[0]= NAN;
-  dt_dcraw_adobe_coeff((const char (*))camera_makermodel, (float(*)[12])XYZ_CAM);
-  if(!isnan(XYZ_CAM[0]))
+  if(!isnan(adobe_XYZ_to_CAM[0][0]))
   {
-  for(int k= 0; k < 9; k++)  m[k] = round(XYZ_CAM[k] * 10000.0f);
-  den = 10000;
+    for(int k= 0; k < 3; k++)
+      for(int i= 0; i < 3; i++)
+        m[k*3+i] = roundf(adobe_XYZ_to_CAM[k][i] * ADOBE_COEFF_FACTOR);
+    den = 10000;
```

Comment:
```suggestion
        m[k*3+i] = roundf(adobe_XYZ_to_CAM[k][i] * ADOBE_COEFF_DENOM);
    den = ADOBE_COEFF_DENOM;
```

Reply:
I keep the name `ADOBE_COEFF_FACTOR` as it is used with * and seems a better term for it.